remove-exec-depend/remove_exec_depend.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import re
+from pathlib import Path
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--package_xml", required=True)
+    parser.add_argument("--exclude_packages", required=False, nargs="*", type=str)
```

Comment:
Seeing [GNU Coding Standards](http://www.gnu.org/prep/stan
dards/html_node/Command_002dLine-Interfaces.html), it's better to use `dash-case`.
http://www.gnu.org/prep/standards/html_node/Option-Table.html#Option-Table

```suggestion
    parser.add_argument("--package-xml", required=True)
    parser.add_argument("--exclude-packages", required=False, nargs="*", type=str)
```

Reply:
I fixed.